static/.htaccess

```diff
@@ -10,6 +10,23 @@ Options -Indexes -MultiViews
 
 RewriteRule ^docs/components/optimize/userguide/combined-reports/(.*)$ /docs/components/optimize/userguide/combined-process-reports/$1 [R=301,L]
 
+RewriteRule ^docs/guides/introduction-to-camunda-platform-8/(.*)$ /docs/guides/introduction-to-camunda/$1 [R=301,L]
+RewriteRule ^docs/guides/introduction-to-camunda-platform-8$ /docs/guides/introduction-to-camunda/ [R=301,L]
+RewriteRule ^docs/8.0/guides/introduction-to-camunda-platform-8/(.*)$ /docs/8.0/guides/introduction-to-camunda/$1 [R=301,L]
+RewriteRule ^docs/8.0/guides/introduction-to-camunda-platform-8$ /docs/8.0/guides/introduction-to-camunda/ [R=301,L]
+RewriteRule ^docs/1.3/guides/introduction-to-camunda-cloud/(.*)$ /docs/1.3/guides/introduction-to-camunda/$1 [R=301,L]
+RewriteRule ^docs/1.3/guides/introduction-to-camunda-cloud$ /docs/1.3/guides/introduction-to-camunda/ [R=301,L]
+RewriteRule ^docs/1.2/guides/introduction-to-camunda-cloud/(.*)$ /docs/1.2/guides/introduction-to-camunda/$1 [R=301,L]
+RewriteRule ^docs/1.2/guides/introduction-to-camunda-cloud$ /docs/1.2/guides/introduction-to-camunda/ [R=301,L]
+RewriteRule ^docs/1.1/guides/introduction-to-camunda-cloud/(.*)$ /docs/1.1/guides/introduction-to-camunda/$1 [R=301,L]
+RewriteRule ^docs/1.1/guides/introduction-to-camunda-cloud$ /docs/1.1/guides/introduction-to-camunda/ [R=301,L]
+RewriteRule ^docs/1.0/guides/introduction-to-camunda-cloud/(.*)$ /docs/1.0/guides/introduction-to-camunda/$1 [R=301,L]
+RewriteRule ^docs/1.0/guides/introduction-to-camunda-cloud$ /docs/1.0/guides/introduction-to-camunda/ [R=301,L]
+RewriteRule ^docs/0.26/guides/introduction-to-camunda-cloud/(.*)$ /docs/0.26/guides/introduction-to-camunda/$1 [R=301,L]
+RewriteRule ^docs/0.26/guides/introduction-to-camunda-cloud$ /docs/0.26/guides/introduction-to-camunda/ [R=301,L]
+RewriteRule ^docs/0.25/guides/introduction-to-camunda-cloud/(.*)$ /docs/0.25/guides/introduction-to-camunda/$1 [R=301,L]
+RewriteRule ^docs/0.25/guides/introduction-to-camunda-cloud$ /docs/0.25/guides/introduction-to-camunda/ [R=301,L]
```

Comment:
Are these redirects needed? All of the docs changed in this PR had their _`id`_ changed, but none had their `slug` change. The `id` is the one used by docusaurus to know where to route links, but the `slug` is the one that users see in their browser.

Reply:
You may be right here @pepopowitz -- I'll remove and see if it still build fine. 👍

Reply:
Built clean. Thanks for the tip @pepopowitz ! @akeller In reference to [this comment](https://github.com/camunda/camunda-platform-docs/pull/1105#discussion_r940700209), Steve and I talked today and I'm going to go ahead and launch a PR to test and possibly drop any excess redirects with slugs like this. I might also go through and label the sections of redirects according to why they're there (C8 migration, "overview" naming changes, etc) as a sort of "htaccess clean up".

Reply:
I _love_ this! Please add some comments in there!